openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -109,4 +106,13 @@ jQuery(function () {
         import(/* webpackChunkName: "editions-table" */ './editions-table')
             .then(module => module.initEditionsTable());
     }
+    // Enable any carousels in the page:
+    // Live NodeList is cast to static array to avoid infinite loops
+    Array.prototype.slice.call(
```

Comment:
Can we use `Array.from`? Does babel rewrite that to something es5 compatible?

Reply:
Nope Babel only gives us syntax not es6 functions. We would need to use shims.

Reply:
:'(

Reply:
It's all good. Im sure this will soon become a pattern and we can abstract it away

I could use jquery instead if you prefer but i was trying to break us out of thr jquery habit

Reply:
To be honest I think the jQuery might be better; the loss in readability isn't worth it (especially since jquery is used profusely throughout the codebase).

Reply:
Done.